docs/tutorial/6-viewsets-and-routers.md

```diff
@@ -116,7 +116,7 @@ Here's our re-wired `snippets/urls.py` file.
 
     # The API URLs are now determined automatically by the router.
     urlpatterns = [
-        url(r'^', include(router.urls))
+        path('', include(router.urls)),
```

Comment:
`path` should also be imported then instead of `url` - sorry for missing this earlier.

You can add `[ci skip]` to not trigger a CI run.

Reply:
My bad, where would I add the [ci skip]?

Reply:
Damn, just realized I should've included it in the commit message, already committed...

Reply:
I'm not 100% sure, but I don't think non-members/collaborators can issue a `[ci skip]`.